src/application/page3/IndividualForm.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import { TextInput } from 'hds-react';
+import { useTranslation } from 'react-i18next';
+import { useForm } from 'react-hook-form';
+import {
+  Address,
+  Application,
+  ContactPerson,
+  FormType,
+} from '../../common/types';
+import { TwoColumnContainer } from '../../component/common';
+import RadioButtons from './RadioButtons';
+import EmailInput from './EmailInput';
+import BillingAddress from './BillingAddress';
+import Buttons from './Buttons';
+
+type Props = {
+  activeForm: FormType;
+  setActiveForm: (id: FormType) => void;
+  application: Application;
+  onNext: () => void;
+};
+
+const IndividualForm = ({
+  activeForm,
+  setActiveForm,
+  application,
+  onNext,
+}: Props): JSX.Element | null => {
+  const { t } = useTranslation();
+
+  const { register, handleSubmit } = useForm({
+    defaultValues: {
+      contactPerson: { ...application.contactPerson },
+      billingAddress: { ...application.billingAddress },
+    },
+  });
+
+  const onSubmit = (data: Application): void => {
+    // todo create copy and edit that
+    // eslint-disable-next-line
+    application.applicantType = 'individual';
+    if (!application.contactPerson) {
+      // eslint-disable-next-line
+      application.contactPerson = {} as ContactPerson;
+    }
+    Object.assign(application.contactPerson, data.contactPerson);
+
+    if (!application.billingAddress) {
+      // eslint-disable-next-line
+      application.billingAddress = {} as Address;
+    }
+
+    // eslint-disable-next-line
+    application.organisation = null;
+    Object.assign(application.billingAddress, data.billingAddress);
+    onNext();
```

Comment:
Maybe the reassignments could be substituted by making a copy of application.

Reply:
yeah, I agree that's why there's the todo comment :) Will refactor that later.